src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`styles/main.scss`). Ensure that your build process (e.g., using Parcel) correctly compiles SCSS to CSS, as browsers do not natively support SCSS files.